src/core/main.go

```diff
@@ -234,6 +235,9 @@ func main() {
 	}
 
 	log.Infof("Version: %s, Git commit: %s", version.ReleaseVersion, version.GitCommit)
+
+	log.Info("Fix empty subiss for meta info data.")
+	oidc.FixEmptySubIss(orm.Context())
```

Comment:
if auth_mode = "OIDC", then fix?

Reply:
Though this problem impact oidc auth mode only but we should try to fix data inconsistency any way regardless the auth mode.